### ISSUE TITLE
aio tests are flaky

### ISSUE THREAD

**Issue by IgorMinar** [angular/angular#14347]
The `aio` build on CI keeps on failing occasionally due to a flake. We need to investigate and fix this.

Example build: https://travis-ci.org/angular/angular/builds/199319022

in case the job gets restarted and the log is lost the relevant part is here:

```
webpack: Compiled successfully.

  site App
    ✗ should show features text after clicking "Features"
      - Expected '' to contain 'Progressive web apps'.

**************************************************
*                    Failures                    *
**************************************************

1) site App should show features text after clicking "Features"
  - Expected '' to contain 'Progressive web apps'.

Executed 1 of 1 spec (1 FAILED) in 14 secs.
[19:29:54] I/launcher - 0 instance(s) of WebDriver still running
[19:29:54] I/launcher - chrome #01 failed 1 test(s)
[19:29:54] I/launcher - overall: 1 failed spec(s)
[19:29:54] E/launcher - Process exited with error code 1
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.


The command "./scripts/ci-lite/build.sh && ./scripts/ci-lite/test.sh" exited with 1.
store build cache
```

**Comment by petebacondarwin** [comment]
@IgorMinar is this still happening? I don't see it.

**Comment by petebacondarwin** [comment]
I think this is fixed (by #14356)
